operator-wg/whitepaper-community-review/README.md

```diff
@@ -0,0 +1,882 @@
+
+The controller will constantly compare the desired state with the current state using the reconciliation loop which ensures that the watched objects get transitioned to the desired state in a defined way.
+
+The operational (or domain-specific) knowledge is usually encapsulated in one or more Kubernetes custom resources which are defined by custom resource definitions.
+
+#### Kubernetes controllers
+A Kubernetes Controller takes care of routine tasks to ensure the desired state expressed by a particular resource type matches the real-world state (current state,https://engineering.bitnami.com/articles/a-deep-dive-into-kubernetes-controllers.html, https://fntlnz.wtf/post/what-i-learnt-about-kubernetes-controller/). For instance, the Deployment controller takes care that the desired amount of pod replicas is running and a new pod spins up, when one pod is deleted or fails.
+
+Technically, there is no difference between a typical controller and an operator. Often the difference referred to is the operational knowledge that is included in the operator. As a result, a controller which spins up a pod when a custom resource is created and the pod gets destroyed afterwards can be described as a simple controller. If the controller has additional operational knowledge like how to upgrade or remediate from errors, it is an operator.
+
+#### Custom resources and custom resource definitions
+Custom resources are used to store and retrieve structured data in Kubernetes (https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/). 
+In the case of an operator, a custom resource contains the required state of the application or domain-specific knowledge but does not contain the implementation logic. Such information could be the version information of application components, but also enabled features of an application or information where backups of the application could be part of this. A custom resource definition (CRD) defines how such an object looks like, for example, which fields exist and how the CRD is named. Such a CRD can be scaffolded using tools (as the operator SDK) or be written by hand.
+
+The following example illustrates, how such an custom resource instance definition could look like:
+
+```
+apiVersion: example-app.appdelivery.cncf.io/v1alpha1
+ kind: ExampleApp
+ metadata:
+   name: appdelivery-example-app
+ spec:
+   appVersion: 0.0.1
+   features:
+     exampleFeature1: true
+     exampleFeature2: false
+   backup:
+     enabled: true
+     storageType: “s3”
+     host: “my-backup.example.com”
+     bucketName: “example-backup”
+ status:
+   currentVersion: 0.0.1
+   url: https://myloadbalancer/exampleapp/
+   authSecretName: appdelivery-example-app-auth
+   backup:
+     lastBackupTime: 12:00
+```
+
+This example creates a custom resource with the name “appdelivery-example-app” of the kind “ExampleApp”.
```

Comment:
`This example creates` I would put `describes`  - as it will be created by i.e. `kubectl apply` command

Reply:
hm, reading further, it has `status`, so it means it is object from running cluster, so maybe `represents` fit better?